vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SpaceManager.java

```diff
@@ -179,6 +180,13 @@ public ISpace getFullyLoadedSpace(String id) {
         return spaces;
     }
 
+    @Override
+    public List<ISpace> getPublishedSpaces() {
```

Comment:
I would make this method generic so that it takes a space status as a parameter.